### DISCUSSION TITLE
Figma-export doesn't work on windows. Is it okay?

### DISCUSSION THREAD

**Discussion by spectosha** [RedMadRobot/figma-export#97] [Q&A] [answered]
We installed figma-export in our projects on mac and windows. And it's doesn't work on last one.
The error when we start command from terminal.

```
"figma-export" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.
```

**Comment by subdan** [comment, marked as answer]
FigmaExport doesn’t support Windows. It build with Swift and SPM. When Apple ports SPM to Windows I will try to add Windows support.
